src/ClientData/CallstackDataTest.cpp

```diff
@@ -200,4 +200,124 @@ TEST(CallstackData, FilterCallstackEventsBasedOnMajorityStartExcludesFunctionToS
                                                                          event4, event5, event6}));
 }
 
+TEST(CallstackData, ForEachCallstackEventOfTidInTimeRangeDiscretized) {
+  CallstackData callstack_data;
+
+  const uint64_t cs_id = 12;
```

Comment:
Just wondering. Why are some literal-initialized constants const and some constexpr?

Reply:
It was like that before, I've  just copy pasted the previous existent tests.

Reply:
I made some changes, PTAL.